react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -370,16 +372,29 @@ export default class JabraService extends VendorImplementation {
     this.changeConnectionStatus({ isConnected: true, isConnecting: false });
   }
 
+  async deviceHasPermissions (deviceLabel: string): Promise<boolean> {
+    const allowedHIDDevices = await (window.navigator as any).hid.getDevices();
+    let deviceFound;
+    allowedHIDDevices.forEach(device => {
+      if (deviceLabel.includes(device?.productName?.toLowerCase())) {
+        deviceFound = true;
+        return;
+      } else {
+        deviceFound = false;
+        return;
```

Comment:
I'm not sure if both or either of the `return`s here are necessary but they are here for the time being.